server/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/temporalio/ui-server/v2
 
-go 1.20
+go 1.21
+
+toolchain go1.22.4
```

Comment:
Definitely don't want this change in here. Same with changes in `server/go.sum`.

```suggestion
go 1.20
```

Reply:
I don't know how I did that. Thanks for catching this.

Reply:
@GraceGardner looks like changes to `server/go.sum` still need to be removed.